data/human/human missions.txt

```diff
@@ -6849,3 +6849,318 @@ mission "FW Refugees to Humanika"
+				die
+
+	npc
+		personality heroic nemesis vindictive launching
+		government "Pirate"
+		ship "Quicksilver" "Dog's Leash"
+		ship "Quicksilver" "Burning Whip"
+		ship "Quicksilver" "Iron Chains"
+		on disable
+			set "adelita: killed pirates"
+
+	on visit
+		dialog `Your escort carrying the spy drones was left behind. Better take off and wait for them to enter the system.`
+
+mission "Adelita 5"
+	name "Return Adelita to <planet>"
+	description "Bring everyone back to <destination>. (You will be paid <payment> after you complete the entire job.)"
+	landing
+	passengers 3
+	cargo "stealth drones" 1
+	"apparent payment" 200000
+	source "Maker"
+	destination "Moonshake"
+	to offer
+		has "Adelita 4: done"
+
+	on offer
+		conversation
+			branch experienced
+				has "adelita: killed pirates"
+
+			`It was close, but you make it back to Maker. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`
+			`	"That was horrible, but we got it done. Can we leave now?" asks Ale.`
+				launch
+
+			label experienced
+			`The pirates were no match for you. Ale and River are soon on board and excited to be out of their disguises, and they spend a lot of time talking with Bunker about his part of the mission. Although exhausted, both are eager to get back and process what they filmed - both emotionally and literally.`
+			`	"That was horrible, but we got it done. Can we leave now?" asks Ale.`
+				launch
+
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+
+	on complete
+		payment 220000
+		log `Helped the underground journalism collective "Adelita Zero" gather video evidence of Syndicate participation in the slave trade. Had to escape pirates while doing so.`
+		conversation
+			`Ale, River, and Bunker look glad to be back on <planet>. "Ugh. Being there in person was bad enough," says Ale, "and now we have to watch it over and over again to edit it into something presentable." She hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you, we really couldn't have done it without you. We'll let you know when it's done. Also, we might need you for some future projects, so expect us to be in touch." The group leaves your ship.`
```

Comment:
```suggestion
			`Ale, River, and Bunker look glad to be back on <planet>. "Being there in person was bad enough," Ale grumble, tapping against a stack of folders on her device. "Now we have to watch it over and over again to edit it into something presentable."`
			
			Before leaving, she hands you a stack of credit chips totaling <payment>. "There's a bonus in there for you, we really couldn't have done it without you and you'll be one of the first to know when it's done. Also, we might be in need of you for some future projects, so expect us to be in touch."`
```

Reply:
The original had a disconnect in emotion. Ale is "telling," or "grumbling" that they had to rewatch the footage, yet somehow, she was somewhat chirpy immediately as she hands the chips.

Reply:
‘Grumbles’, and is there an extra blank line now? 7167 seems to only be a blank line, and 7168 seems to start with a blank line too. Of course that could just be my phone formatting it weirdly.